obs_status.yml

```diff
@@ -97,21 +89,12 @@ bad:
   1092619768: 9
   1105592952: 9
   1108353064: 9
-  1117787424: 9
-  1130649544: 9
   1139171624: 9
   1178221128: 9
-  1197635184: 9
 mags:
 - agasc_id: 5115672
   mag_aca: 11.422
   mag_aca_err: 0.1
-- agasc_id: 33690776
-  mag_aca: 9.984
```

Comment:
So, this one was already in the bad list, but did it have an entry in the mags table already?  And if not, does it deserve one?